src/main/external-resources/UMS.conf

```diff
@@ -866,6 +866,20 @@ ffmpeg_fontconfig =
 # Default: true
 ffmpeg_mencoder_problematic_subtitles =
 
+# Use Libfdk for aac encoding
+# --------------------------------------------
+# Whether FFmpeg should a high quality aac encoder, libfdk. Note: need a build by yourself, due to Licence.
+# to be transcoded.
+# Both engines can handle subtitles, but MEncoder is more stable right now.
```

Comment:
Why is there a note about MEncoder on a setting for FFmpeg? I don't understand what it means